extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMusicSearchExtractor.java

```diff
@@ -105,7 +108,7 @@ public void onFetchPage(@Nonnull final Downloader downloader) throws IOException
                 .end()
                 .value("query", getSearchString())
                 .value("params", params)
-            .end().done().getBytes("UTF-8");
+            .end().done().getBytes(StandardCharsets.UTF_8);
```

Comment:
Is that going to break compatibility with the legacy version?
Or can we use that safely, because this is a precoompiled lib?

Reply:
Oh, I have no idea, I can revert that commit if needed

Reply:
See https://github.com/TeamNewPipe/NewPipeExtractor/pull/269
We should remove the other occurencies of that class, too.

Reply:
There's an issue to provide a backport: https://issuetracker.google.com/issues/169047771

Reply:
Reverted @TobiGr

Reply:
thanks